src/registrar/templates/base.html

```diff
@@ -144,13 +146,16 @@ <h4 class="usa-alert__heading">New domain requests are paused</h4>
   <header class="usa-header usa-header-basic">
     <div class="usa-nav-container">
       <div class="usa-navbar">
-        {% block logo %}
-        <div class="usa-logo" id="extended-logo">
-          <strong class="usa-logo__text" >
-            <a href="{% url 'home' %}"> .gov </a>
-          </strong>
+        <div class="usa-logo-group">
```

Comment:
This container div needs to go. It's not needed, and without its CSS it's now messing up the layout on mobile:

![Screen Shot 2023-12-05 at 8 08 08 PM](https://github.com/cisagov/manage.get.gov/assets/107004823/246fd4b6-27eb-4a8e-86d4-e512bbbb931d)

Reply:
Haha omg not me in Slack being like "yes I will remember to remove!!" and then promptly forgetting 🤦🏼‍♀️ Thank you Rachid 🤣

Reply:
Great catch again!

<img width="376" alt="Screenshot 2023-12-05 at 5 24 33 PM" src="https://github.com/cisagov/manage.get.gov/assets/13954664/35ac0146-6e66-4d4d-b915-25fbf5de1fbd">